packages/amplify-codegen/src/commands/statements.js

```diff
@@ -55,15 +55,50 @@ async function generateStatements(context, forceDownloadSchema, maxDepth, withou
 
     try {
       fs.ensureDirSync(opsGenDirectory);
-      generate(schemaPath, opsGenDirectory, {
-        separateFiles: true,
-        language,
-        maxDepth: maxDepth || cfg.amplifyExtension.maxDepth,
+      const schemaData = loadSchema(schemaPath);
+      const generatedOps = generateGraphQLDocuments(schemaData, {
+        maxDepth: maxDepth || cfg.amplifyExtension.maxDepth
       });
+      await writeGeneratedStatements(language, generatedOps, opsGenDirectory, true);
       opsGenSpinner.succeed(constants.INFO_MESSAGE_OPS_GEN_SUCCESS + path.relative(path.resolve('.'), opsGenDirectory));
     } finally {
       opsGenSpinner.stop();
     }
   }
 }
+
+async function writeGeneratedStatements(language, generatedStatements, outputPath, separateFiles) {
+  const fileExtension = FILE_EXTENSION_MAP[language];
+  if(!generatedStatements) {
+    return;
```

Comment:
Should this throw an error instead of clean return?

Reply:
This file is subject to change in upcoming PR. I've made some edits to get e2e passing but am going to re-visit this.